### DISCUSSION TITLE
GM COMMAND BLIZZARD

### DISCUSSION THREAD

**Discussion by Seigneur-fantome** [azerothcore/azerothcore-wotlk#5958] [Guides & Tips]
hello anyone, i am french, and today, i post this information.

you will know how use "wishper" as moderator BLIZZARD.

create macro with this command :
(english version)
/run SendChatMessage ("Hi" ..WhoFrame.selectedName .. ", i am" ..UnitName ("player") .. ", gm. Do you have any time for a request?", "WHISPER", nil, WhoFrame.selectedName)

(french version)
/run SendChatMessage("salutations "..WhoFrame.selectedName..", je suis "..UnitName("player")..", Maître du jeu. Auriez vous un instant à me consacrer ?" , "WHISPER", nil, WhoFrame.selectedName)

result (french only)
https://cdn.discordapp.com/attachments/815935097227968524/844677644305694760/unknown.png

**Comment by Kitzunu** [comment]
Cool, I suggest GMGenie as well, it is a good addon to add macro texts :D

**Reply by Seigneur-fantome** [reply]
Yes. But. Gmgeni have problem with ticket. In background. Gmgenie generate many command in consol.

And. Now. You Know how use it. Without gmgeni
